build.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(unused_extern_crates)]
```

Comment:
Why is this needed?

Reply:
It was needed in order to get it to compile with the particular version of Rust that I was building with. It might be that it was an older or newer version where it didn't like the `extern crate cc` declaration. I'll remove it and force-push.